vault_core/src/vault.rs

```diff
@@ -154,6 +154,7 @@ pub struct Vault {
 impl Vault {
     pub const MAX_REWARD_DELTA_BPS: u16 = 50; // 0.5%
     pub const MIN_WITHDRAWAL_SLIPPAGE_BPS: u16 = 50; // 0.5%
+    pub const INITIALIZATION_TOKEN_AMOUNT: u64 = 10_000;
```

Comment:
Make this a pass into initalize_vault and then in docs we give suggestions for a min amount

Reply:
Done